json_schemas/framework-agreement-details.json

```diff
@@ -3,6 +3,10 @@
   "additionalProperties": false,
   "minProperties": 1,
   "properties": {
+    "countersignerDetails": {
```

Comment:
Does it make sense to call this field `countersignerDetails`, which kind of implies its a dictionary full of information (like `signerDetails`) when at the moment it is just a string for the name of the counter signer. Would something like `countersignerName` be more appropriate?